etc/visual-studio/UnitTests.vcxproj

```diff
@@ -50,6 +50,7 @@
         MBEDTLS_CONFIG_FILE="mbedtls-config.h";
         OPENTHREAD_CONFIG_FILE="openthread-windows-config.h";
         OPENTHREAD_MULTIPLE_INSTANCE;
+        OPENTHREAD_FTD=1;
```

Comment:
Looks like you could put the `OPENTHREAD_FTD=1` in the header file and just put the SPI and UART defines in the project files that need them. I don't care too much either way though.